### PR TITLE
Support definition of public frontend attributes and only serialize those

### PR DESCRIPTION
**Pull request by michalczaplinski**
Fixing #13

### CODE REVIEW

File: src/gutenberg-packages/wordpress-blocks.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import { InnerBlocks, useBlockProps } from "@wordpress/block-editor";
 import { registerBlockType as gutenbergRegisterBlockType } from "@wordpress/blocks";
+import { getFrontendAttributes } from "./utils";
```
Comment (michalczaplinski) [inline src/gutenberg-packages/wordpress-blocks.js:3]: I've put the `getFrontendAttributes` in a separate file for no reason other than that it's easier to mock in tests this way.

File: src/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { getFrontendAttributes } from "../gutenberg-packages/utils";
+
+jest.mock("@wordpress/blocks", () => {
+  return {
+    __esModule: true,
+    getBlockType: () => ({
```
Comment (michalczaplinski) [inline src/__tests__/index.js:6]: We mock this function which is being called inside of the `getFrontendAttributes()` to give us a our fake attributes.

**Review by cbravobernal — APPROVED** [review]
LGTM!

**Review by luisherranz — APPROVED** [review]
It even has tests 😱

**Pull request merged**